pom.xml

```
@@ -10,18 +10,18 @@
 
     <name>MCAuthLib</name>
     <description>A library for authentication with Minecraft accounts.</description>
-    <url>http://github.com/Steveice10/MCAuthLib/</url>
+    <url>http://github.com/GeyserMC/MCAuthLib/</url>
 
     <scm>
-        <connection>scm:git:git@github.com:Steveice10/MCAuthLib.git</connection>
-        <developerConnection>scm:git:git@github.com:Steveice10/MCAuthLib.git</developerConnection>
-        <url>git@github.com:Steveice10/MCAuthLib/</url>
+        <connection>scm:git:git@github.com:GeyserMC/MCAuthLib.git</connection>
+        <developerConnection>scm:git:git@github.com:GeyserMC/MCAuthLib.git</developerConnection>
+        <url>git@github.com:GeyserMC/MCAuthLib/</url>
     </scm>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <jdk.version>1.7</jdk.version>
-        <argLine></argLine>
+        <argLine/>
```

Gonna presume this wasn't meant to change, revert for a cleaner commit and makes it easier for people that need to change it later.
```suggestion
        <argLine></argLine>
```